### DISCUSSION TITLE
Question about the policy configuration files

### DISCUSSION THREAD

**Discussion by Doomnometron** [Sleepw4lker/TameMyCerts#8] [Q&A] [answered]
Hi there,

do I need to create a policy configuration file for *every* certificate template that exists on the CA or only for the templates, that I want to tame?

Like I have five templates and I only need to tame one template, do I have to create five xml files or only one?

I find the statements somehow contradictory, therefore the question. Sorry about that.
"Create a policy file in XML format for **each certificate template you want to apply a policy** for in the folder you specified during installation."
https://github.com/Sleepw4lker/TameMyCerts/blob/main/user-guide/configuring.adoc

"**Für jede Zertifikatvorlage muss eine gleichnamige Datei** im zuvor erstellten und während der Installation angegebenen Ordner erzeugt werden."
https://www.gradenegger.eu/?p=17181

Danke für den Dienst an der Community!

**Comment by Sleepw4lker** [comment, marked as answer]
Hello, thanks for reaching out. You only need a policy configuration file for each certificate template you actually want to apply a policy to. If no policy configuration file exists for a certificate template, the request will be handled like the policy module wouldnt be installed (only the logic of the Windows Default policy will get applied).
